packages/kbn-unified-data-table/src/components/data_table_column_header.tsx

```diff
@@ -105,12 +105,14 @@ export const DataTableTimeColumnHeader = ({
   dataView,
   dataViewField,
   headerRowHeight = 1,
+  columnLabel,
 }: {
   dataView: DataView;
   dataViewField?: DataViewField;
   headerRowHeight?: number;
+  columnLabel?: string;
 }) => {
-  const timeFieldName = dataViewField?.customLabel ?? dataView.timeFieldName;
+  const timeFieldName = columnLabel || (dataViewField?.customLabel ?? dataView.timeFieldName);
```

Comment:
following previous comment, if we pass columnLabel={columnDisplayName} to this component, 
 this logic can be simplified to be:

```js
const timeFieldName = columnLabel || dataView.timeFieldName;
```

As the `dataView.customLabel` checking will be done already by the parent component.

Reply:
are we the only ones that utilizes customLabel field ?
if I removed that, won't that break other team previous implementation ?

Reply:
wont that break other people implementation if they use .customLabel field ?